### ISSUE TITLE
Change signature of AddRoleToUserAsync

### ISSUE THREAD

**Issue by nbarbettini** [okta/okta-sdk-dotnet#102]
Adding a role to a user needs to take one parameter, `type`:

```
POST {{url}}/api/v1/users/{{userId}}/roles
{
    "type": "{{roleType}}"
}
```

**Comment by nbarbettini** [comment]
Blocked on https://github.com/okta/openapi/issues/82

**Comment by HakanL** [comment]
Blocked by a non-existing (404) issue?

**Comment by nbarbettini** [comment]
@HakanL It's showing up as 404 because it's a private repository, sorry.

Is this method something you are actively trying to use?

**Comment by HakanL** [comment]
Ah, I see. Not really, I was trying to research another issue and it led me to this, figured I would just report that the link was "broken".

**Comment by nbarbettini** [comment]
Gotcha, thanks! I was curious since I haven't seen a lot of people asking for this particular API yet. If it was blocking you, I'd try to bump it up in priority. We'll get it fixed eventually. 👍 

Cheers!
